Playback/Core/Scene/SceneThink_Playback.asm

```diff
@@ -1,13 +1,40 @@
 ################################################################################
 # Address: 801a6348
 ################################################################################
-.include "Common/Common.s"
+.include "Online/Core/EXIFileLoad/TransferFile.asm"
 .include "Playback/Playback.s"
 
 .set REG_Floats, 30
 .set REG_BufferPointer, 29
-.set REG_Text,28
-.set REG_FrameCount,27
+.set REG_Text, 28
+.set REG_FrameCount, 27
+.set REG_LOCAL_DATA_ADDR, 25
+.set REG_CAM_GOBJ, 22
+.set REG_LOGO_JOBJ, 21
+.set REG_SLPLOGO, 19
+.set REG_LOGO_GOBJ, 18
+
+# symbol offsets
+.set SLPLOGO_LOGO_JOBJDESC, 0x0
+.set SLPLOGO_CAMDESC, 0x4
+.set COBJ_LINKS, 0x24
+.set LOGO_GXLINK, 9
+
+  bl DATA_BLRL
+  mflr REG_LOCAL_DATA_ADDR
+  b FBegin
+
+
+DATA_BLRL:
+  blrl
+# File related strings
+.string "slpLogo.dat"
```

Comment:
@JordanLozinski could you reuse the existing slippi dat files? 
Looks cumbersome to have a whole .dat file just for the logo.
The .dat file that has the css chat stuff could share this logo. 

If the current name implies that its only for the css, then I think its fine to just rename it. 
Theres not that much stuff in it for it to be a memory issue, if it becomes one it can just separated in the future as "slpCommons" or something like that.

Reply:
I think it's probably better to have a separate one for different scenes tbh. That's what the game normally does.

Reply:
> I think it's probably better to have a separate one for different scenes tbh. That's what the game normally does.

The game does reuses common dat files, which I can see this one being part of.

I.e: theres a common menu dat file the game uses for different scenes on the main menu, 1p css, vs scenes etc. Cant recall the exact name of it right now though it also does it for some other files.

Reply:
slpLogo is a SOBJ with a JOBJDesc array and a CObjDesc array, and slpCSS is a JOBJDesc, so I'll need to make slpCSS part of the array in slpLogo, and then rewrite and test the stuff that pulls the chat window jobjdesc from the file. I'm reluctant to potentially break or compromise the chat functionality for a small benefit. 

However we can rename slpLogo to slpCommons or something and use it for miscellaneous jobjs, camdescs, lights, etc. in the future